src/functions-and-arrays.js

```diff
@@ -1,38 +1,127 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+
+
+function maxOfTwoNumbers(harrySize, bobSize) {
+  if (harrySize < bobSize) {
+    return bobSize;
+  } 
+  
+  else if (bobSize < harrySize) {
+    return harrySize;
+   }
+
+   else  {
+     return harrySize;
+   }
+  
+};
+
+let harrySize = 7;
+let bobSize = 5;
+
+console.log(maxOfTwoNumbers(harrySize, bobSize));
+
 
 
 
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+//const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+
+/*findLongestWord("mystery brother aviator crocodile pearl orchard crackpot"); */
+
+
+const str = "mystery brother aviator crocodile pearl orchard crackpot";
+
+const myArray = str.split(" ");
+
+
+
+
+
+
+
+
+
+
+function longestStringForLoop(myArray) {
+  let longestWord = " ";
+ 
+  for(var i = 0; i < myArray.length; i++){
+    if(myArray[i].length > longestWord){
+	longestWord = myArray[i].length;
+     }
+  }
+  return longestWord;
+}
+
+console.log(longestStringForLoop(myArray));
```

Comment:
Here you have changed the name of the function - so the tests cannot find and check it! This week more labs will have automated tests, so be sure not to touch the function ma,e only write code inside the function scope :)

Also, your longestWord has to be a string, the actual word. Currently, on line 51, your saving (and later returning) the length of that string.